iaso/models/forms.py

```diff
@@ -109,6 +114,10 @@ class Form(SoftDeletableModel):
     def latest_version(self):
         return self.form_versions.order_by("-created_at").first()
 
+    @property
```

Comment:
can you leave documentation on what this is supposed to represent becasue that's not clear to me.

Reply:
@olethanh can you do it and merge this PR if ok for you please ?

Reply:
Well it's hard to document what I don't understand.
Since @mathvdh  is continuing on this feature probably the best would be  that he see if he want to reuse this code and close this PR (or set it as draft) ?

Reply:
I didn't dig into the details yet, but yes I think we can put this issue back in draft or close it

Reply:
I put it back in draft